src/services/messaging.js

```diff
@@ -0,0 +1,45 @@
+const { StatusCodes } = require('http-status-codes');
+const { generateOperationOutcome } = require('../utils/fhir');
+
+function processMessage(req, res) {
+  const messageBundle = req.body;
+  const messageHeader = messageBundle.entry[0].resource;
+
+  // Validate message
+  if (!messageBundle || !messageHeader) {
```

Comment:
If `messageBundle` is null or isn't the expected structure then an exception would have already been thrown when trying to set `messageHeader` above, so these lines should be reworked a little bit. Not that we expect these to happen in the real world often.

Also for some reason the exception response I got (below) was in HTML.
```
<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8">
	<title>Error</title>
</head>

<body>
	<pre>TypeError: Cannot read property &#39;0&#39; of undefined<br> &nbsp; &nbsp;at processMessage (/Users/dehall/medmorph-backend/src/services/messaging.js:6:44)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/Users/dehall/medmorph-backend/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at next (/Users/dehall/medmorph-backend/node_modules/express/lib/router/route.js:137:13)<br> &nbsp; &nbsp;at Route.dispatch (/Users/dehall/medmorph-backend/node_modules/express/lib/router/route.js:112:3)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/Users/dehall/medmorph-backend/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at /Users/dehall/medmorph-backend/node_modules/express/lib/router/index.js:281:22<br> &nbsp; &nbsp;at Function.process_params (/Users/dehall/medmorph-backend/node_modules/express/lib/router/index.js:335:12)<br> &nbsp; &nbsp;at next (/Users/dehall/medmorph-backend/node_modules/express/lib/router/index.js:275:10)<br> &nbsp; &nbsp;at Function.handle (/Users/dehall/medmorph-backend/node_modules/express/lib/router/index.js:174:3)<br> &nbsp; &nbsp;at router (/Users/dehall/medmorph-backend/node_modules/express/lib/router/index.js:47:12)</pre>
</body>

</html>
```